src/gain.js

```diff
@@ -124,8 +124,13 @@ define(function (require) {
   };
 
   /**
-   *  Set the output level of the gain node.
-   *
+   *  Set the output level of the GainNodes.Gain is the amount of amplification done to the signal. 
+   *  The main way to affect the loudness of a sound is using a Web Audio node called GainNodes. 
```

Comment:
Please clarify that it's the Web Audio API `GainNode` (no "s" at the end). If mentioning the underlying Web Audio API components, it could be nice to link to their documentation on MDN.

Reply:
Could you please explain a little bit what did you actually mean by "linking to their documentation on MDN ".

Reply:
Resolved :+1: